css/mobile.css

```diff
@@ -28,9 +28,9 @@ header {
   flex-direction: column;
 }
```

Comment:
# This is the best work so far 👍 but I noticed your code was not passed the linter test. I would recommend you lint your code before you would submit 🤞